omero/conf.py

```diff
@@ -107,20 +107,6 @@
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path.extend(['themes'])
 
-# -- Options for LaTeX output --------------------------------------------------
```

Comment:
I would leave the template in place, but empty, as I did here: https://github.com/ome/ome-model/blob/master/docs/sphinx/conf.py#L304

Then the template structure is in place as for all the other bits of conf.py we don't use.